metacat-metadata/src/functionalTest/java/com/netflix/metacat/metadata/store/data/repositories/CrdbDataMetadataRepositoryTests.java

```diff
@@ -0,0 +1,18 @@
+//CHECKSTYLE:OFF
```

Comment:
What's the rationale for opting the tests out of checkstyle?

Reply:
Out checkstyle policies already exclude the test package just from an ease of development perspective I guess. we need to update them to exclude the functionalTest package as well.